src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -13,6 +13,18 @@ def test_clear():
     assert actual == expected
 
 
+# In the set data test, we test for these cases.
+# (1) x and y are non-empty array values, and res is positive (the most generic case)
```

Comment:
Good. This test should cover what is desired for the other attributes that are determined from these inputs.